spec/cobra_commander/executor_spec.rb

```diff
@@ -21,18 +21,9 @@
     expect(output.string).to match %r{===> node_manifest \(.*spec/fixtures/app/node_manifest\)}
   end
 
-  it "executes with clean environment" do
-    output = StringIO.new
-    CobraCommander::Executor.exec(umbrella.components, "env", output)
+  it "cleans BUNDLE environment so a nested bundle call can succeed" do
+    expect(Bundler).to receive(:with_original_env).and_yield
 
-    expect(output.string).to match %r{===> a \(.*spec/fixtures/app/components/a\)\n^$}
```

Comment:
Why don't you need these assertions any more?

Reply:
They were asserting the environment is clean, they're no longer valid as the environment could have something.